class JobError(Exception):
	pass

class FailedJobError(JobError):
	pass
